src/main/kotlin/pt/up/fe/ni/website/backend/service/ErrorMessages.kt

```diff
@@ -0,0 +1,11 @@
+package pt.up.fe.ni.website.backend.service
+
+object ErrorMessages {
+    fun postNotFound(postId: Long): String {
+        return "post not found with id $postId"
+    }
+
+    fun projectNotFound(id: Long): String {
+        return "project not found with id $id"
+    }
+}
```

Comment:
This is already pretty cool but I wonder if we can move every error message to the resources. What do you think?